.moai/specs/SPEC-V3R3-ARCH-007/acceptance.md

```
@@ -0,0 +1,210 @@
+
+# CHANGELOG entry
+grep "BC-V3R3-006" CHANGELOG.md && echo "OK: BC-V3R3-006 documented" || echo "MISSING"
+```
+
+Maps to: REQ-ARCH007-008
+
+---
+
+## Edge Cases
+
+### EC-1: runtime.yaml 부재
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)